refnx/reduce/batchreduction.py

```diff
@@ -427,3 +590,52 @@ def run_list(entry, mode='refl'):
 
     # valid = [int(r) for r in l if not np.isnan(r)]
     return [int(v) for v in valid]
+
+
+class BatchReducerConfiguration(dict):
+    """ Configuration for the batch reducer
+
+    Example:
```

Comment:
We need an example of a configuration file, and what key:value are allowed. Otherwise I'd have no idea.

Reply:
Also, is there a point to having a batch reduction configuration if in the future we have configuration files for each reflect:direct configuration. 
Off the top of my head the only setting one needs is whether to remove redundant points once two datasets have been spliced.